src/ui/display_nbgl.c

```diff
@@ -204,6 +212,37 @@ void ui_accept_transaction_flow(void) {
     nbgl_refresh();
 }
 
+void ui_accept_selftransfer_flow(void) {
```

Comment:
I think here we could try to avoid code duplication by moving all of the code except the tagValue filling to a dedicated function :)

Reply:
Solved in 33f2e13 by adding an `is_self_transfer` parameter, so I could instead delete both `ui_validate_selftransfer` and `ui_accept_selftransfer_flow`.